applications/tari_console_wallet/src/automation/commands.rs

```diff
@@ -609,6 +621,22 @@ pub async fn command_runner(
                     eprintln!("DiscoverPeer error! {}", e);
                 }
             },
+            BurnFunds(args) => {
+                match burn_tari(
```

Comment:
I think these two names should match - `BurnFunds` vs. `burn_tari`

Reply:
Definitely ! Thanks for the suggestions